setup.py

```diff
@@ -19,12 +19,13 @@
     install_requires=[
         'docutils',
         'sphinxcontrib-napoleon>=0.5.1',
-        'funcsigs;python_version<"3"',
+        'funcsigs;python_version<"3.3"',
```

Comment:
I don't plan to use them either, but that's just a more convenient way to say "function signatures were introduced in 3.3"